### DISCUSSION TITLE
Webcam selection

### DISCUSSION THREAD

**Discussion by VegaDeftwing** [noisedeck/noisedeck-docs#17] [Ideas]
I have two traditional webcams and a Leap Motion (which identifies as a webcam) on my system. Currently, it seems all the browsers I've tried (Vivaldi, Chromium, Firefox) do default to the right camera; however, I'm not sure that this is anything beyond luck as, far as I remember, I never actually set the default camera to be used?

Looking at https://webcamtests.com it does look like adding a camera select to the page should be possible.

This may also allow for having mult-cam inputs in the future with two different all-seeing-eye modules

**Comment by jesshewitt** [comment]
Good idea. Will add to feature requests.
